lib/rules/indent.js

```diff
@@ -850,7 +850,18 @@ module.exports = {
 
             ExportNamedDeclaration(node) {
                 if (node.declaration === null) {
-                    addElementListIndent(getTokensAndComments(node).slice(1), node.specifiers, 1);
+                    const tokensInNode = getTokensAndComments(node);
+                    const closingCurly = sourceCode.getLastToken(node, astUtils.isClosingBraceToken);
+                    const closingCurlyIndex = lodash.sortedIndexBy(tokensInNode, closingCurly, token => token.range[0]);
```

Comment:
I don't think this would affect performance because both methods use binary search anyway, so they would start at the middle of the array. I think `sortedLastIndexBy` is only useful if there are multiple occurrences of a given value in a sorted array.